runbotics/runbotics-orchestrator-ui/src/src-app/components/Tile/ProcessTile/ProcessTile.tags.styles.ts

```diff
@@ -0,0 +1,96 @@
+import ExpandLessIcon from '@mui/icons-material/ExpandLess';
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+import { Chip, Typography } from '@mui/material';
+import styled from 'styled-components';
+
+export const StyledChip = styled(Chip)`
+    margin: 0 8px 8px 0;
```

Comment:
there's a risk of margin overlapping. imo wrap original Chip with div with padding.

Reply:
and try equally distributed spacing